src/cluster/websocket/WebSocket.cc

```diff
@@ -372,6 +372,30 @@ void WebSocketEventDispatcher::Process(const WebSocketSubscribeFinished& fin) {
         return;
     }
 
+    if ( ! entry.ready_to_publish ) {
```

Comment:
Just FTR, nothing to do here.

Like I wrote over Slack, it is slightly weird that websocket layer keeps track of whether a backend is ready to publish while the backend would accept publishes without error even if it is not (currently backends only know about clients when they subscribe to _something_, and while Zeek nodes typically subscribe, we don't expect all websocket clients would). If we would clean this up in the future we would probably add some way for a client to announce itself to the backend which could then keep track of its readiness.